Dockerfile

```diff
@@ -9,7 +9,7 @@ ADD . /gopath/src/github.com/drone/drone/
 WORKDIR /gopath/src/github.com/drone/drone
 
 RUN apt-get update
-RUN apt-get -y install zip libsqlite3-dev sqlite3 1> /dev/null 2> /dev/null
+RUN apt-get -y install zip libsqlite3-dev sqlite3 >/dev/null 2>&1
```

Comment:
then what about using the even shorter syntax instead `&>/dev/null` 

Source from [bash manpage](http://www.gnu.org/software/bash/manual/bash.html#Redirecting-Standard-Output-and-Standard-Error)

> 3.6.4 Redirecting Standard Output and Standard Error
>
> This construct allows both the standard output (file descriptor 1) and the standard error output (file descriptor 2) to be redirected to the file whose name is the expansion of word.
>
> There are two formats for redirecting standard output and standard error:
>
>     &>word
>
> and
>
>     >&word
>
> Of the two forms, the first is preferred. This is semantically equivalent to
>
>     >>word 2>&1
>